cmd/protoc-gen-gorums/gengorums/gorums_func_map.go

```diff
@@ -106,6 +109,12 @@ var funcMap = template.FuncMap{
 	"isOneway": func(method *protogen.Method) bool {
 		return hasMethodOption(method, gorums.E_Multicast, gorums.E_Unicast)
 	},
+	"isAsync": func(method *protogen.Method) bool {
+		return hasMethodOption(method, gorums.E_Async)
+	},
+	"isConfigurationCall": func(method *protogen.Method) bool {
+		return hasMethodOption(method, gorums.E_Multicast, gorums.E_Quorumcall, gorums.E_Correctable, gorums.E_Async)
+	},
 	"methods": func(services []*protogen.Service) (methods []*protogen.Method) {
```

Comment:
Perhaps we can remove this one; it does not appear to be used. I don't remember why I added it, and it seems we are using `.Methods` instead.